### PR TITLE
Add french translations

### PR DESCRIPTION
**Pull request by MDLC01**

### CODE REVIEW

**Comment by Marmare314** [comment]
Thanks! I will add this to the 0.1.1 release.

File: src/translations.typ [hunk 1]
```diff
@@ -19,6 +19,16 @@
...
+  "fr": (
+    "theorem": "Théorème",
+    "lemma": "Lemme",
+    "corollary": "Corollaire",
+    "remark": "Remarque",
+    "proposition": "Proposition",
+    "example": "Exemple",
+    "definition": "Définition",
+    "proof": "Preuve"
+  ),
```
Comment (porcaror) [inline src/translations.typ:31]: "it":  (
    "theorem":   "Teorema",
    "lemma":   "Lemma",
    "corollary":   "Corollario",
    "remark":  "Osservazione",
    "proposition":  "Proposizione",
    "example":  "Esempio",
    "definition":   "Definizione ",
    "proof":   "Dimostrazione"
),

Comment (porcaror) [reply src/translations.typ:31]: This is the Italian version

Comment (MDLC01) [reply src/translations.typ:31]: You should make a pull request instead: fork the repository, create a new branch, add italian translations and make a pull request.

Comment (Marmare314) [reply src/translations.typ:31]: Yes that would be the best way, but I can also add them for you in case you don't want to deal with that.

**Pull request merged**